.github/workflows/lint-just-fmt.yml

```diff
@@ -0,0 +1,46 @@
+# Make sure justfile source code is consistently formatted.
+
+name: "lint-just-fmt.yml"
+
+on:
+  pull_request: {}
+  merge_group:
+    types: ["checks_requested"]
```

Comment:
Can we run this just on PRs that change the `justfile`? Either using the workflow syntax, or using a dedicated Action if we plan on marking this workflow as required.